draft-ietf-tls-esni-13.md

```diff
@@ -0,0 +1,1846 @@
+---
```

Comment:
You should probably nuke this file.

Reply:
Thanks -- didn't even notice that I pushed this. I'll remove it.